scripts/app-release/chore-release.py

```diff
@@ -0,0 +1,381 @@
+            return tags.split("\n")[0]
+        return None
+    except subprocess.CalledProcessError:
+        return None
+
+
+def count_commits_since_tag(repo_path: Path, tag: str) -> int:
+    """Count commits between tag and HEAD."""
+    try:
+        count = run_command(
+            ["git", "rev-list", f"{tag}..HEAD", "--count"],
+            cwd=repo_path,
+        )
+        return int(count)
+    except (subprocess.CalledProcessError, ValueError):
+        return 0
+
+
+def increment_tag(last_tag: str, tag_type: str) -> str:
+    """
+    Increment a tag based on its type.
+
+    - semver:  v1.19.6 → v1.19.7 (increments patch)
+    - alpha:
+        - v8.8.0-alpha.6 → v8.8.0-alpha.7 (increments alpha number)
+        - v8.8.0         → v8.9.0-alpha.0 (increments minor, starts alpha series)
+    - numeric:
+        - v67 → v68 (increments the integer after 'v')
+    """
+    if tag_type == "semver":
+        match = re.match(r"^v(\d+)\.(\d+)\.(\d+)$", last_tag)
+        if not match:
+            raise ValueError(f"Cannot parse semver tag: {last_tag}")
+        major, minor, patch = match.groups()
+        new_patch = int(patch) + 1
+        return f"v{major}.{minor}.{new_patch}"
+
+    elif tag_type == "alpha":
+        alpha_match = re.match(r"^(v\d+\.\d+\.\d+-alpha\.)(\d+)$", last_tag)
+        if alpha_match:
+            prefix, alpha_num = alpha_match.groups()
+            new_alpha_num = int(alpha_num) + 1
+            return f"{prefix}{new_alpha_num}"
+
+        semver_match = re.match(r"^v(\d+)\.(\d+)\.(\d+)$", last_tag)
+        if semver_match:
+            major, minor, _patch = semver_match.groups()
+            new_minor = int(minor) + 1
+            return f"v{major}.{new_minor}.0-alpha.0"
+
+        raise ValueError(f"Cannot parse tag for alpha increment: {last_tag}")
+
+    elif tag_type == "numeric":
+        m = re.match(r"^v(\d+)$", last_tag)
+        if not m:
+            raise ValueError(f"Cannot parse numeric tag: {last_tag}")
+        n = int(m.group(1)) + 1
+        return f"v{n}"
+
+    else:
+        raise ValueError(f"Unknown tag_type: {tag_type}")
+
+
+def parse_chore_release_branch(name: str) -> Optional[Tuple[int, int, int]]:
+    """Parse 'chore_release-X.Y.Z' → (X, Y, Z) as ints, or None if no match."""
+    m = re.match(r"^chore_release-(\d+)\.(\d+)\.(\d+)$", name)
+    if not m:
+        return None
+    major, minor, patch = map(int, m.groups())
+    return major, minor, patch
+
+
+def find_latest_chore_release_branch(clone_url: str) -> str:
+    """
+    Clone opentrons repo to a temp dir, list all remote chore_release-* branches,
+    and return the one with the highest X.Y.Z version.
+    """
+    from tempfile import TemporaryDirectory
+
+    print("Finding latest chore_release-* branch from opentrons...")
+
+    with TemporaryDirectory() as tmpdir:
+        tmp_path = Path(tmpdir)
+        run_command(
+            ["git", "clone", "--origin", "origin", clone_url, "tmp-opentrons"],
```

Comment:
```suggestion
            ["git", "clone", clone_url, "tmp-opentrons"],
```
uses the default origin name anyway